src/main/java/net/minecraftforge/server/command/CommandSetDimension.java

```diff
@@ -75,6 +73,12 @@ public Entity placeEntity(Entity entity, ServerWorld currentWorld, ServerWorld d
                 repositionedEntity.setPositionAndUpdate(pos.getX(), pos.getY(), pos.getZ());
                 return repositionedEntity;
             }
+            
+            @Override
+            public PortalInfo getPortalInfo(Entity entity, ServerWorld destWorld, Function<ServerWorld, PortalInfo> defaultPortalInfo)
+            {
+                return new PortalInfo(entity.getPositionVec(), Vector3d.ZERO, entity.rotationYaw, entity.rotationPitch);
+            }
```

Comment:
I am guessing this is required to make it work properly and otherwise it just doesn't teleport (instead of how it currently just silently null pointers)?

Reply:
It doesn't throw a crash, but the game will lock up since the vanilla method would be used here by default. I'm not exactly sure where it locks up, but it does.

Reply:
I figured this was the case I just wanted to confirm there wasn't being added due to some breaking change being introduced.